scripts/README.md

```diff
@@ -0,0 +1,255 @@
+# scripts
+
+The TANF app uses several scripts through its lifecycle. 
+These don't all get used by or interacted with by us too often,
+but some are mission critical during deployment and review of 
+the application. When a developer is working on these scripts, 
+they should update this documentation so future developer's 
+understand the role of the scripts
+
+# set-backend-env-vars.sh 
+
+## usage 
+
+```bash
+
+./scripts/set-backend-env-vars.sh CGAPPNAME_BACKEND CG_SPACE
+```
+
+## args
+
+`CGAPPNAME_BACKEND` the sub domain of the app you are trying to set up environment variables for.
```

Comment:
Might be useful to provide some real world examples - ie `tdp-backend-raft`, `tanf-dev`, etc

Reply:
Yes, agreed, please provide a concrete example. Could you also please link to our other documentation which lists out all the spaces and backends? Like [this one](https://github.com/raft-tech/TANF-app/blob/c0c9423dcd4d9b87930eb655a74dd8f2701e3dcf/docs/Technical-Documentation/TDP-environments-README.md) for the backends.